boefjes/boefjes/plugins/kat_cve_finding_types/schema.json

```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Arguments",
+    "type": "object",
+    "properties": {
+        "PORTS": {
+            "title": "CVEAPI_URL",
+            "maxLength": 2048,
+            "type": "string",
+            "description": "URL of the CVE API, defaults to https://cve.openkat.dev/v1"
```

Comment:
You could add the "default" label to show the user which url will be the default. We might not render it yet, but we will.